classes/patterns/class-action.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Action pattern class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * Class Action.
+ *
+ * @package P4GBKS\Patterns
+ */
+class Action extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/action-pattern-layout';
+	}
+
+	/**
+	 * Returns the pattern config.
+	 *
+	 * @param array $params Optional array of parameters for the config.
+	 */
+	public static function get_config( $params = [] ): array {
+		$media_link = esc_url( get_template_directory_uri() ) . '/images/placeholders/placeholder-546x415.jpg';
+
+		return [
+			'title'      => __( 'Action', 'planet4-blocks-backend' ),
+			'categories' => [ 'layouts' ],
+			'content'    => '
+				<!-- wp:group {"className":"block"} -->
+					<div class="wp-block-group">
+						<!-- wp:media-text {"mediaLink":"' . $media_link . '","mediaType":"image"} -->
+							<div class="wp-block-media-text is-stacked-on-mobile">
+								<figure class="wp-block-media-text__media">
+									<img src="' . $media_link . '" alt="' . __( 'Default image', 'planet4-blocks-backend' ) . '"/>
+								</figure>
+								<div class="wp-block-media-text__content">
+									<!-- wp:gravityforms/form /-->
+								</div>
+							</div>
+						<!-- /wp:media-text -->
+						<!-- wp:group {"backgroundColor":"grey-05","align":"full","className":"is-style-large-padding"} -->
+							<div class="wp-block-group alignfull has-grey-05-background-color has-background is-style-large-padding">
+								<!-- wp:group {"className":"container"} -->
+									<div class="wp-block-group container">
+										' . SideImageWithTextAndCta::get_config(
+										[
+											'media_position'    => 'right',
+											'title_placeholder' => __( 'The problem', 'planet4-blocks' ),
+										]
+									)['content'] . '
+									</div>
+								<!-- /wp:group -->
+							</div>
+						<!-- /wp:group -->
+						<!-- wp:planet4-blocks/covers {
```

Comment:
If you change to something like [this](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/884/files#diff-25f0e4e3db0b77616a6f7b7662e425dc8343525cac2a6b8a478461a2ef08b376R86-R88) it would fix the issue we have with this block. 
We should definitely follow up this issue and find the best solution because it happens not only with the Cover but also with other `p4/blocks`, i.e. the Carousel Header.

Reply:
[Done](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/879/commits/f793f2921f8b39a0401651a29bcdf56ae6d1bd95)! Indeed it's not an ideal solution, but it seems to work